### PR TITLE
rm: Add a data retention policy doc

### PR DESCRIPTION
**Pull request by doanac**
Signed-off-by: Andy Doan <andy@foundries.io>

## Readiness

* [x] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

_Why merge this PR? What does it solve?_

## Checklist

_Optional. Add a 'x' to steps taken._
_You can fill this out after opening the PR. "Did I..."_

* [ ] Run spelling and grammar check, preferably with linter.
* [x] Avoid changing any header associated with a link/reference.
* [ ] Step through instructions (or ask someone to do so).
* [ ] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [ ] Match tone and style of page/section.
* [x] Run `make linkcheck`.
* [x] View HTML in a browser to check rendering.
* [x] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [ ] follow best practices for commits.
  * [ ] Descriptive title written in the imperative.
  * [ ] Include brief overview of QA steps taken.
  * [ ] Mention any related issues numbers.
  * [x] End message with sign off/DCO line (`-s, --signoff`).
  * [x] Sign cSPHINXBUILD=./venv/bin/sphinx-build makeommit with your gpg key (`-S, --gpg-sign`).
  * [ ] Squash commits if needed.
* [x] Request PR review by a technical writer and at least one peer.

## Comments

_Any thing else that a maintainer/reviewer should know._
_This could include potential issues, rational for approach, etc._

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 358be1675a915122bc54d0a3b2eea34bc1433ab6 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1546/docs/artifacts/html/index.html

File: source/reference-manual/factory/data-retention.rst [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+Device data includes things such as:
+
+ * Hardware information from the lshw_ tool.
+ * Details of the last 10 OTAs (``fioctl updates show``)
+ * When the device was created
+ * The Factory user that created the device
+ * The device's local IPv4 address. *Depends on deployment details, this is normally a class C IP Address and not the public IPv4 address it accesses the Internet from*
+ * The device's MAC address
+
+The service and data is inside Google's GCP us-central1-a region.
```
Comment (kprosise) [inline source/reference-manual/factory/data-retention.rst:99]:
```suggestion
Both service and data gets kept inside Google's GCP us-central1-a region.
```

File: source/reference-manual/factory/data-retention.rst [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+The data is periodically backed up to Google Storage.
+Backups over 6 months old are pruned.
```
Comment (kprosise) [inline source/reference-manual/factory/data-retention.rst:101]:
```suggestion
Backups over 6 months old get pruned.
```

**Review by kprosise — APPROVED** [review]
I did a handful of trimming, while hopefully not changing the meaning. I also attempted to clarify/make explicit some points.

Overall I treated this how I would any other documentation, and was not as pedantic as a more legal document would likely be.

I try to avoid referencing "customers", though this would be difficult in some cases.  This also brought to mind the larger issue of  Customer, User, member, owner, and any other way we reference those utilizing FoundriesFactory. But that is something for another day, and feel free to ignore or edit any of my suggestions related to this.

Also, I want to thank you for using semantic line breaks, I think this would have taken much longer to review otherwise!

**Review by caiotpereira — APPROVED** [review]

File: source/reference-manual/factory/data-retention.rst [hunk 3]
```diff
@@ -0,0 +1,109 @@
+.. _ref-data-retention:
+
+Data Retention Policies
+=======================
+
+Users often need to understand data retention policies within a FoundriesFactory so that they can work through compliance processes for their product.
```
[the remark this answers is not in the document]
Comment (doanac) [reply source/reference-manual/factory/data-retention.rst:6]: how about "your product" instead of "a product"?

**Comment by doanac** [comment]
Docs for d14b938578fa2e3264b99ff5fcfb7678c1b10d16 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1549/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 367fc316d3915a82af11458e0e8521a1683de128 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1550/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 0272fc8dc5483170b908b91d052df5b988fe465a are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1551/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 94254c3093e896a959f284c0596f66b4a270a29b are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1552/docs/artifacts/html/index.html

File: source/reference-manual/factory/data-retention.rst [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+The storage bucket is multi-regional in the United States.
+The database is periodically backed up to Google Storage.
+Backups over 6 months old are pruned.
+
+CI Workers
+~~~~~~~~~~
+CI work takes place in ephemeral instances (Docker containers) that are removed upon the completion of a CI Run.
+The exception to this is the `sstate cache`_ for LmP builds.
+This is kept on an NFS drive in the customer's CI region.
+This data is periodically pruned to remove old data and completely deleted when a Factory is deleted.
```
[the remark this answers is not in the document]
Comment (doanac) [reply source/reference-manual/factory/data-retention.rst:52]: no. it depends on the rate of change within factories and how fast their data is becoming unused.

File: source/reference-manual/factory/data-retention.rst [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+ * The Factory's TUF Metadata
+ * Device data
+
+Device data includes things such as:
+
+ * Hardware information from the lshw_ tool.
+ * Details of the last 10 OTAs (``fioctl updates show``)
+ * When the device was created
+ * The Factory user that created the device
+ * The device's local IPv4 address. *Depends on deployment details, this is normally a class C IP Address and not the public IPv4 address it accesses the Internet from*
```
[the remark this answers is not in the document]
Comment (doanac) [reply source/reference-manual/factory/data-retention.rst:96]: i'll commit it and fix it if its not rendering well

File: source/reference-manual/factory/data-retention.rst [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+
+The storage bucket is multi-regional in the United States.
+The database is periodically backed up to Google Storage.
+Backups over 6 months old are pruned.
+
+CI Workers
+~~~~~~~~~~
+CI work takes place in ephemeral instances (Docker containers) that are removed upon the completion of a CI Run.
+The exception to this is the `sstate cache`_ for LmP builds.
+This is kept on an NFS drive in the customer's CI region.
```
[the remark this answers is not in the document]
Comment (doanac) [reply source/reference-manual/factory/data-retention.rst:51]: its multi-tenant. we group customers into clusters that share a drive

**Comment by doanac** [comment]
Docs for 985d084e35cc856994b0fdaa84ed2c985c643480 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1555/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for fee8994fe95482a2a858bf4fb4699e4127dbd94e are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1556/docs/artifacts/html/index.html

**Pull request merged**